server/endpoints.go

```diff
@@ -33,8 +33,8 @@ const (
 	OrganizationsEndpoint = "organizations"
 	// ReportEndpoint returns report for provided {organization}
 	ReportEndpoint = "organizations/{org_id}/clusters/{cluster}/users/{user_id}/report"
-	// RuleEndpoint returns rule report for provided {organization} {cluster} and {rule_id}
-	RuleEndpoint = "organizations/{org_id}/clusters/{cluster}/users/{user_id}/report/{rule_id}"
+	// SingleRuleEndpoint returns rule report for provided {organization} {cluster} and {rule_id}
```

Comment:
Why this was renamed?

Reply:
I just wanted to keep the same name for endpoints which are 2 parts of the same thing(here and in proxy). If you want, I can keep previous version, I don't like it, but it can be subjective thing.

Reply:
`RuleEndpoint` will be enough.

Reply:
Reverted back to `RuleEndpoint`